app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    todo, file_from, file_to, *_ = command.split() + [None] * 3
+    if todo != "mv" or None in (file_from, file_to):
+        print("Usage: mv SOURCE DEST")
+        return
+
+    if not os.path.exists(file_from):
+        print("File not found:", file_from)
+        return
+
+    if file_to.endswith("/"):
+        file_to = os.path.join(file_to, os.path.basename(file_from))
+
+    dir_path_to = os.path.dirname(os.path.abspath(file_to))
+    dir_path_from = os.path.dirname(os.path.abspath(file_from))
+
+    if not os.path.exists(dir_path_to):
+        os.mkdir(dir_path_to)
+
+    if dir_path_from == dir_path_to:
+        os.rename(file_from, file_to)
+        return
+
+    with open(file_from, "r") as file_stream_file_from:
```

Comment:
what does _file_stream_file_from_ variable mean? (файл_потік_файл_з)🧐
because it's hard for me to understand🥲

better to use this name  _stream_file_from_